### DISCUSSION TITLE
Unistall NonSteamLaunchers

### DISCUSSION THREAD

**Discussion by Kaabvoo** [moraroy/NonSteamLaunchers-On-Steam-Deck#29] [Q&A] [answered]
is there a way to unistall the actual application? Or I just need to remove the filees and thats all (files from the comptdata)?

**Comment by moraroy** [comment, marked as answer]
@Kaabvoo there is no "application" to remove. That's what the "Start Fresh" or "Uninstall" buttons do. Everything that is needed is downloaded in you Downloads folder and then once the script is finished it is deleted. There is nothing left over besides what is installed that is necessary. Proton GE and the launchers you choose.

**Reply by Kaabvoo** [reply]
Cool! thanks

**Reply by patridge** [reply]
Neither the Uninstall nor the Start Fresh buttons seem to do anything for me. All the launchers I previously installed are still present in Gaming Mode, but not working. Since it sounds like I should install the Decky Loader and its version now, I wanted to uninstall the existing NonSteamLaunchers first. Unfortunately, that isn't working. Is there some manual way to uninstall all the launchers and the NonSteamLaunchers stuff itself?

Everything was working fine a month or two ago, at least within Epic. This week, the installed launchers all just fail to run. For that issue, I tried shutting down the NSLGameScanner as suggested in #436, but that didn't fix it. The only "launcher" that works is Xbox Game Pass.

**Comment by marinajb** [comment]
My I already delete from comptdata folder, I already removed from my library this thing keep adding and come back to my steam os, how I remove this from entire from Steam OS??

**Reply by moraroy** [reply]
does Start Fresh not work?

**Reply by marinajb** [reply]
I didnt had the decky loader version, I had the desktop version. Now I installed the decky loader version and removed unistall, and now stopped adding the desktop version to my library.

**Reply by moraroy** [reply]
@marinajb the desktop version has a magnifying glass 🔍 . This will stop the scanner. Next time open up the NonSteamLaunchers.desktop file and you will see many options to choose from. "Start Fresh" will wipe all of NSL essentially uninstalling it.
